### DISCUSSION TITLE
Deleteable on specific uploaded files

### DISCUSSION THREAD

**Discussion by zeeshantariq08** [filamentphp/filament#9687] [Ideas]
i have multiple files that are being uploaded, but I want on some of them deletable false , right now i am unable to do

**Comment by sinnbeck** [comment]
Curious how you will determine which should be deletable and which should not? 

Filament uses filepond for uploads, so perhaps start by looking at their docs. You might need to make your own version of the file uploader to add extra features :)

**Reply by zeeshantariq08** [reply]
My report is associated with workflow, once the report is approved then reopen it so before reopening i have to lock all the files to not be deleted, and after reopening the new file will be uploaded and will be deleted, hope I have explained well.
